cloud/pkg/admissioncontroller/admission.go

```diff
@@ -144,133 +149,171 @@ func configTLS(opt *options.AdmissionOptions, restConfig *restclient.Config) (*t
 
 // registerWebhooks registers the admission webhook.
 func (ac *AdmissionController) registerWebhooks(opt *options.AdmissionOptions, cabundle []byte) error {
-	ignorePolicy := admissionregistrationv1beta1.Ignore
-	failPolicy := admissionregistrationv1beta1.Fail
-	deviceModelCRDWebhook := admissionregistrationv1beta1.ValidatingWebhookConfiguration{
+	ignorePolicy := admissionregistrationv1.Ignore
+	failPolicy := admissionregistrationv1.Fail
+	noneSideEffect := admissionregistrationv1.SideEffectClassNone
+
+	// DeviceModel webhook
+	deviceModelCRDWebhook := admissionregistrationv1.ValidatingWebhookConfiguration{
```

Comment:
Why split one ValidatingWebhookConfiguration to multi ValidatingWebhookConfiguration

Reply:
you mean keep only one `ValidatingWebhookConfiguration` and contain multi `MutatingWebhook` in that one `ValidatingWebhookConfiguration` ?

Reply:
fixed, i misunderstood the config name, it's `validate-devicemodel`, i thought one config for one CRD. change the name to `validate-configure`

Reply:
yes，the name is not good, but if we just update the name,  for user who upgrade the webhook, there maybe two ValidatingWebhookConfiguration, the old one and the new one, it may execute twice for one crd

Reply:
oh, will revert it and add a TODO to make it more compatible

Reply:
Fixed, please help review again